src/services/token.service.js

```diff
@@ -0,0 +1,37 @@
+const { Token } = require('../models/Token.js');
+
+async function save(userId, refreshToken) {
+  const token = await Token.findOne({
+    where: { userId },
+  });
+
+  if (token) {
+    token.refreshToken = refreshToken;
+
+    await token.save();
+
+    return;
```

Comment:
The `return` statement here is used to exit the function early if a token is found and updated. While this works, it might be clearer to explicitly return a value indicating the operation's success, such as `return true;`, and `return false;` after creating a new token. This can help with debugging and understanding the function's flow.